open-company/open-company-web
# Fix empty drafts

Bug: empty state for drafts mode is currently broken in dark mode, also it uses the wrong icon.
Here is the latest design: https://share.goabstract.com/bf2d36e1-2e61-465a-b8ca-94b12b336e01?mode=design&sha=a7f5114491b8e1d909e6638bad7eb7095afef709

To test:
- check drafts section when empty
- check drafts section when empty in dark mode
- check drafts mode when NOT empty
- check drafts section when NOT empty in dark mode
